plank/services/stock-ticker-service.go

```diff
@@ -227,26 +220,14 @@ func queryStockTickerAPI(symbol string) (map[string]interface{}, error) {
 		return nil, err
 	}
 
-	// Alpha Vantage which is the provider of this API limits API calls to 5 calls per minute and 500 a day, and when
-	// the quota has been reached it will return a message in the Note field.
-	if len(tickerData.Note) > 0 {
-		return nil, fmt.Errorf(tickerData.Note)
-	}
-
-	if tickerData == nil || tickerData.TimeSeries == nil {
-		return nil, err
-	}
-
-	// extract the data we need.
-	latestClosePriceStr := tickerData.TimeSeries[tickerData.MetaData.LastRefreshed]["4. close"].(string)
-	latestClosePrice, err := strconv.ParseFloat(latestClosePriceStr, 32)
-	if err != nil {
-		return nil, err
-	}
-
 	return map[string]interface{}{
-		"symbol":        symbol,
-		"lastRefreshed": tickerData.MetaData.LastRefreshed,
-		"closePrice":    latestClosePrice,
+		"symbol":             symbol,
+		"lastRefreshed":      time.Unix(tickerData.LastUpdated, 0).String(),
+		"currentPrice":       tickerData.CurrentPrice,
+		"change":             tickerData.Change,
+		"highDayPrice":       tickerData.HighDayPrice,
+		"lowDayPrice":        tickerData.LowDayPrice,
+		"openPrice":          tickerData.OpenPrice,
+		"previousClosePrice": tickerData.PreviousClosePrice,
```

Comment:
Can you make this backwards compatible by including `closePrice` in this return please.

Reply:
done